### ISSUE TITLE
Accessibility: Name, Role, Value (WCAG 4.1.2)

### ISSUE THREAD

**Issue by veler** [Microsoft/pxt#2364]
Standard:
All interactive components must be should be programmatically determined, which means that we must defines the role, value and names of those components.
http://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html

Current situation :
Some components like the setting menu and help menu must have as a description that it is a button, it controls a menu, and it is collapsed.
In a more general way, all important components to the user must be described and must explain how to interact with them.

**Comment by veler** [comment]
All those points are part of [https://github.com/Microsoft/pxt/pull/2446](https://github.com/Microsoft/pxt/pull/2446)
